[quant] Re-enable fold_convbn in quantize_script

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32379 [WIP][quant][graphmode] FoldConvBatchNorm2d support shared ClassTypes
* #32543 [WIP][quant][graphmode] ConvBn Folding works with bias=None
* #32380 [quant][graphmode][refactor] Better API for fold_convbn
* #32374 [refactor] Adding FoldConvBatchNorm2dHelper
* #32303 [quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script
* **#32302 [quant] Re-enable fold_convbn in quantize_script**
* #32206 [quant] Re-enable test_nested that has different qconfig for shared ClassType
* #32205 [quant][graphmode] Support quantizing shared ClassType with different qconfigs

Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D19508035](https://our.internmc.facebook.com/intern/diff/D19508035)

jerryzh168 committed:
[quant] Re-enable fold_convbn in quantize_script

Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 9c45c55f:


**None of the build failures appear to be your fault**.

* **1/1** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=9c45c55f120f62dc11237ddbf69020a5ff89056e).

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4273112) pytorch_xla_linux_xenial_py3_6_clang7_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87201312) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87201312)) :snowflake:
<details>
<summary>
<code>$<3>$<3> f6122d7a68cb: Pushing  3.058GB/8.765GB $<3>$<3> f6122d7a68cb: Pushing  8.765GB/8.765GB toomanyrequests: Rate exceeded</code>
</summary>

```
$<3>$<3> a7e2c767a8ce: Preparing   
$<3>$<3> 79b6ec083c25: Preparing   
$<3>$<3> 66c8fe4ff6be: Preparing   
a7e2c767a8ce: Waiting  $<3>$<3> 66c8fe4ff6be: Waiting   
$<3>$<3> 191ab0b78c7f: Preparing   
$<3>$<3> 6a764e8f2e62: Preparing   
$<3>$<3> 136e021b5828: Preparing   
$<3>$<3> 77008e118980: Preparing   
$<3>$<3> 6cb741cb00b7: Preparing   
77008e118980: Waiting  $<3>$<3> 6cb741cb00b7: Waiting   
f6122d7a68cb: Pushing  3.058GB/8.765GB $<3>$<3> f6122d7a68cb: Pushing  8.765GB/8.765GB toomanyrequests: Rate exceeded 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

jerryzh168 committed:
Update on "[quant] Re-enable fold_convbn in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Re-enable fold_convbn in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Re-enable fold_convbn in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
👍 
PS: Please try to keep such commits close to the changes that allow to do them and include in the description what was broken before and how it has been fixed now.

jerryzh168 committed:
Update on "[quant] Re-enable fold_convbn in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Re-enable fold_convbn in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Re-enable fold_convbn in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Re-enable fold_convbn in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D19508035](https://our.internmc.facebook.com/intern/diff/D19508035)

[ghstack-poisoned]
